.github/workflows/_lint_py.yaml

```diff
@@ -99,4 +99,6 @@ jobs:
 
           trap remove_matcher EXIT
 
-          mypy .
+          mypy
```

Comment:
I dont know what this do. I have only seen mypy used along with some path (like in the next line). I googled but did not find anything.

Reply:
When you don't provide a path/package/module, it checks the configuration file (in fairseq2 it is pyproject.toml) to figure out the default setting. I updated pyproject.toml in this PR to explicitly list src, fairseq2n/python, and tests as the paths it should check.

Reply:
Thank you :)